app/bundles/CoreBundle/Assets/js/6.forms.js

```diff
@@ -548,7 +552,8 @@ Mautic.createOption = function (value, label) {
 }
 
 /**
- * Updates operator select and value input format based on selected field and operator
+ * Updates operator select and value input format based on selected field and
+ * operator
```

Comment:
This change looks a bit random 😄 
```suggestion
 * Updates operator select and value input format based on selected field and operator
```